sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -12,7 +12,7 @@
     </PackageReleaseNotes>
     <Description>
       This client library enables working with the Microsoft Azure Storage Blob service for storing binary and text data.
-      For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/Changelog.txt
+      For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md
       in addition to the breaking changes https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt
```

Comment:
Should this also make BreakingChanges.txt into BreakingChanges.md?

Reply:
Since its not named Changelog it did not think there was need to change it since it won't be enforced by the tool either ways.

Reply:
@tg-msft I think we should instead put the breaking changes into the changelog at the version they occur and we could highlight them there.

Reply:
I'm fine with that.  Would you guys mind updating the changelog guidance to include recommendations on how we should make breaks vs. other updates clear in the log?

Reply:
>I don't think anybody's going to find our changelog by clicking that. We don't link them in the README.md files so they'd have to navigate to the files view and start clicking around.

I plan to update our README.md template to fix patterns with busted links and I will add a link to the CHANGELOG in that template and I will also add a CHANGELOG template with the format and information to include like the breaking changes. 

As for what is displayed on nuget.org we should work out exactly what to put but I think we should include a link the the changelog and a copy of the most recent entry for each package release. That link will be based on a sha so it will be stable.

Reply:
Sorry, I don't understand how the link for "For this release see notes" at https://www.nuget.org/packages/Azure.Storage.Blobs would be based on a sha?  I can see how Heath's suggestion of using a non-yet-existing tag would work, but not a sha.

Reply:
It will be similar to how release notes are in this [package](https://www.nuget.org/packages/Azure.Messaging.EventHubs/5.0.0-preview.6) but it will be the release SHA version. Similar to what's done for readmes [here](https://github.com/Azure/azure-sdk-for-net/blob/a8607eff533f14218bf0db13d9a24023915e3075/eng/Directory.Build.Data.targets#L20)